backend/routes.py

```diff
@@ -1,12 +1,13 @@
 from flask import Blueprint
 from flask import jsonify
 
-import backend.db.client as db_client
+from .db import client as db_client
 
 blueprint = Blueprint('api', __name__)
 
 
 @blueprint.route('/classrooms')
 def classrooms():
```

Comment:
Lets rename this endpoint and function `recycling_data` and return all data in the db.